keywords.robot

```diff
@@ -1733,14 +1733,10 @@ Check That USB Devices Are Detected
     ...    assumes that a bootable image is hosted on the PiKVM already.
     [Arguments]    ${boot_menu}
 
-    IF    '${DUT_CONNECTION_METHOD}' == 'pikvm'
-        Should Contain Match    ${boot_menu}    *PiKVM*
-    ELSE
-        @{attached_usb_list}=    Get Current CONFIG List Param    USB_Storage    name
-        FOR    ${stick}    IN    @{attached_usb_list}
-            # ${stick} should match with one element of ${boot_menu}
-            Should Contain Match    ${boot_menu}    *${stick}*
-        END
+    @{attached_usb_list}=    Get Current CONFIG List Param    USB_Storage    name
```

Comment:
@JanPrusinowski As @krystian-hebel has noticed before, when removing this pikvm branch (which we agreed to on the call), we must update the documentation string as well.

Reply:
Removed in: 7cde5cbb39b651db442620c425fe257f341b2e39